### PR TITLE
Add doc plugin

### PR DESCRIPTION
**Pull request by Manon-Oreins**
Doc added for the plugin math, possible to see the ReadTheDocs in http://inginious-problems-math.readthedocs.io/

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+import sys
+import os
+sys.path.insert(0, os.path.abspath('../inginious_problems_math/'))
+sys.path.insert(1, os.path.abspath('../base-containers/base/'))
```
Comment (anthonygego) [inline doc/conf.py:21]: You should probably configure the documentation project from scratch using sphinx-quickstart, there are several lines in here that are accurate for the main repo but not for this plugin, and then copy the specific settings you want to share between both.

Comment (Manon-Oreins) [reply doc/conf.py]: I've done that and changed it in the next commit just now

**Comment by anthonygego** [comment]
LGTM. Can you add me to the rtd project maintainers ?

**Comment by Manon-Oreins** [comment]
Done

**Pull request merged**